dnngior/gapfill_class.py

```diff
@@ -528,8 +549,58 @@ def binarySearch(self,
                 #alpha = delta
                 pass
             print('\n\n', 'condition is currently: ', abs(alpha - beta), '\n\n')
-        
+
         # List that has minimum nr of reactions, sum of cost or sum of flux, dependend on output.
-        minimum_set = self.get_minimum(R, R_flux, R_cost, criteria = self.result_selection) 
+        minimum_set = self.get_minimum(R, R_flux, R_cost, criteria = self.result_selection)
 
         return  minimum_set
+
+    def remove_candidates(self, list_to_remove):
+        """
+        Function to manually remove candidates from all_reactions and weights.
+        Input:
+            list of reactions to remove from all reactions
+        Output:
+            None
```

Comment:
That is a bit funny.
Consider having something like "trimmed list of candidate reactions "